src/components/Errors/ResponseError/ResponseError.tsx

```diff
@@ -0,0 +1,11 @@
+import type {IResponseError} from '../../../types/api/error';
+
+interface ResponseErrorProps {
+    error: IResponseError;
+    className?: string;
+    defaultMessage?: string;
```

Comment:
maybe default to something like “Response error”? just to have anything for an empty input